.guiance.clarity.design-app/src/styles/code-highlight.scss

```diff
@@ -0,0 +1,94 @@
+@use '@cds/core/tokens/tokens.scss';
+
+// accessible light theme derived from prism-solarizedlight.css
+// I found compliant colors using https://app.contrast-finder.org/.
+:root {
+  --code-highlight-background: #{tokens.$cds-global-color-white};
+  --code-highlight-base1: #6d797a; // solarizedlight: #93a1a1
+  --code-highlight-base01: #586e75; // solarizedlight: #586e75
+  --code-highlight-blue: #267cb9; // solarizedlight: #268bd2
+  --code-highlight-cyan: #298478; // solarizedlight: #2aa198
+  --code-highlight-green: #767c17; // solarizedlight: #859900
+  --code-highlight-yellow: #887626; // solarizedlight: #b58900
+  --code-highlight-orange: #9d6e04; // solarizedlight: #cb4b16
+}
+
+// accessible dark theme derived from prism-solarizedlight.css
+// I found compliant colors using https://app.contrast-finder.org/.
+[cds-theme~='dark'] {
+  --code-highlight-background: #{tokens.$cds-global-color-black};
+  --code-highlight-base1: #6d797a; // solarizedlight: #93a1a1
+  --code-highlight-base01: #517a8a; // solarizedlight: #586e75
+  --code-highlight-blue: #268bd2; // solarizedlight: #268bd2
+  --code-highlight-cyan: #2aa198; // solarizedlight: #2aa198
+  --code-highlight-green: #859900; // solarizedlight: #859900
+  --code-highlight-yellow: #b58900; // solarizedlight: #b58900
+  --code-highlight-orange: #cb4b16; // solarizedlight: #cb4b16
+}
```

Comment:
Can you try and use the viz tokens instead of hadr coded values.

https://clarity.design/documentation/charts/colors

Reply:
I would rather not. These colors are tightly tuned for color contrast and using the chart colors in a non-chart usage could very easily result in unintended issues. In my opinion, it would be misuse of the aliases to cherry-pick these colors for text.

For example, the closest color might be a "sequential" color, but that would be arbitrary and just a coincidence. I wouldn't be using the alias as intended because this isn't a sequential chart.

Reply:
@kevinbuhmann is there a example page that I can observe those colors?

Reply:
I tried some token based colors. Can you confirm they work?

```suggestion
  :root {
    --code-highlight-background: #{tokens.$cds-alias-object-interaction-background};

    --code-highlight-base1: #{tokens.$cds-global-color-construction-600}; // #6d797a // solarizedlight: #93a1a1
    --code-highlight-base01: #{tokens.$cds-global-color-construction-700}; // #586e75; // solarizedlight: #586e75
    --code-highlight-blue: #{tokens.#cds-global-color-blue-700}; // #267cb9; // solarizedlight: #268bd2
    --code-highlight-cyan: #{tokens.#cds-global-color-jade-700}; // #298478; // solarizedlight: #2aa198
    --code-highlight-green: #{tokens.#cds-global-color-lime-800}; // #767c17; // solarizedlight: #859900
    --code-highlight-yellow: #{tokens.#cds-global-color-yellow-800}; // #887626; // solarizedlight: #b58900
    --code-highlight-orange: #{tokens.#cds-global-color-ochre-800}; // #9d6e04; // solarizedlight: #cb4b16
  }

  // accessible dark theme derived from prism-solarizedlight.css
  // I found compliant colors using https://app.contrast-finder.org/.
  [cds-theme~='dark'] {
    --code-highlight-base1: #{tokens.$cds-global-color-construction-500}; // #6d797a; // solarizedlight: #93a1a1
    --code-highlight-base01: #{tokens.$cds-global-color-construction-400}; // #517a8a; // solarizedlight: #586e75
    --code-highlight-blue: #{tokens.#cds-global-color-blue-600}; // #268bd2; // solarizedlight: #268bd2
    --code-highlight-cyan: #{tokens.#cds-global-color-jade-600}; // #2aa198; // solarizedlight: #2aa198
    --code-highlight-green: #{tokens.#cds-global-color-lime-600}; // #859900; // solarizedlight: #859900
    --code-highlight-yellow: #{tokens.#cds-global-color-yellow-700}; // #b58900; // solarizedlight: #b58900
    --code-highlight-orange: #{tokens.#cds-global-color-tangerine-600}; // #cb4b16; // solarizedlight: #cb4b16
  }
```

Reply:
I updated the code highlight colors to use global tokens based on the above suggestion. I had to adjust some of the colors for contrast. I also updated the clarity.design website. https://github.com/vmware-clarity/website/pull/791

Reply:
Thanks.